opm/simulators/flow/python/PyBlackOilSimulator.hpp

```diff
@@ -40,7 +40,8 @@ class PyBlackOilSimulator
     using Simulator = Opm::GetPropType<TypeTag, Opm::Properties::Simulator>;
 
 public:
-    PyBlackOilSimulator( const std::string& deckFilename);
+    PyBlackOilSimulator(const std::string& deckFilename,
+                        const std::vector<std::string>& args = {});
```

Comment:
Could we drop the default argument? If I understand this correctly the only call site is line 300 in PyBlackOilSimulator.cpp, so it should be fine.

Reply:
strictly speaking line 300 is not a call site, it is defining the python wrapping of the constructor, the call site is in the .py files. but okay, seems pybind does indeed allow default values on python side without having the same on c++ side so i have removed the default value.